src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs

```diff
@@ -263,4 +264,44 @@ public async Task RejectDisputeUpdateRequestAsync(long updateStatusId, Cancellat
         DisputantUpdateRequestRejected message = new(updateStatusId);
         await _bus.PublishWithLog(_logger, message, cancellationToken);
     }
+
+    /// <summary>
+    /// Returns a list of all disputes with pending update requests.
+    /// </summary>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    public async Task<ICollection<Dispute>> GetAllDisputesWithPendingUpdateRequestsAsync(CancellationToken cancellationToken)
+    {
+        ICollection<Dispute> disputes = new Collection<Dispute>();
+        ICollection<TrafficCourts.Common.OpenAPIs.OracleDataApi.v1_0.DisputantUpdateRequest> pendingDisputeUpdateRequests = await _oracleDataApi.GetDisputantUpdateRequestsAsync(null, Status.PENDING, cancellationToken);
+
+        foreach (TrafficCourts.Common.OpenAPIs.OracleDataApi.v1_0.DisputantUpdateRequest disputantUpdateRequest in pendingDisputeUpdateRequests)
+        {
+            if (disputes.FirstOrDefault(x => x.DisputeId == disputantUpdateRequest.DisputeId) is null)
+            {
+                try
+                {
+                    Dispute pendingDispute = await _oracleDataApi.GetDisputeAsync(disputantUpdateRequest.DisputeId, cancellationToken);
+                    disputes.Add(pendingDispute);
+                }
+                catch (Exception ex) {  
+                    // dont crash carry on

```

Comment:
I'm seeing unused ex variable here.
I assume this is a placeholder to catch the 404 exception when the Dispute is not found